packages/mobile/src/exchange/WithdrawCeloScreen.test.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react'
```

Comment:
I wrote in Slack about this and people seem to have different opinions on it, but I wasn't sure if I should use `Celo` or `Gold` to refer to the currency. I used Celo when I started working, so I just kept using that, but I have no strong opinion on this and can change it if anyone has.

Reply:
👍  I agree with using Celo, it will encourage a move away from Gold